.golangci.yaml

```diff
@@ -20,6 +22,11 @@ linters:
               isFirstField: Warn
               usePatchStrategy: Ignore
               useProtobuf: Ignore
+            forbiddenmarkers:
+              markers:
```

Comment:
`kubebuilder:validation:Pattern` as we prefer using CEL expressions?

Reply:
`kubebuilder:validation:UniqueItems` as well? IIRC this marker doesn't actually work right?